components/treeview/stories/treeview.stories.js

```diff
@@ -0,0 +1,241 @@
+// Import the component markup template
+import { Template } from "./template";
+
+export default {
+  title: "Treeview",
```

Comment:
On the docs site, it's called "Tree View", so this should probably be two words to match that.